tests/conn_can/main.c

```diff
@@ -101,7 +102,7 @@ static int _list(int argc, char **argv) {
     return 0;
 }
 
-static int _send(int argc, char **argv)
+static int _send(int argc, char **argv, bool not_rtr)
```

Comment:
Instead of `not_rtr`, I think it would be simpler to just call it `bool rtr`

Reply:
should be ok now.